docs/guides/1230-persistent-cache.md

```diff
@@ -0,0 +1,219 @@
+has a powerful cache system that triggers actions only when it's necessary.
+
+However, sometimes you can't keep a persistent Buildkit daemon along your CI/CD.
+For instance, if you use a GitHub Actions runner, your daemon will be created on each
+run and **cache will be lost**.
+
+In this page, we will see how to use `--cache-from` and `--cache-to` flags to
+keep a permanent cache, from a local environment to GitHub Actions.
+
+## Ephemeral cache
+
+As an example, we will use a Dagger plan to build a Go program.
+
+You can use any Go project and the following snippet to build it in a
+standard way
+
+```cue file=../tests/guides/persistent-cache/build.cue title="build.cue"
+package ci
+
+import (
+  "dagger.io/dagger"
+  "universe.dagger.io/go"
+)
+
+dagger.#Plan & {
+  // Retrieve source from current directory
+  // Input
+  client: filesystem: ".": read: {
+    contents: dagger.#FS
+    // Include only contents we need to build our project
+    // Any other files or patterns can be added
+    include: ["**/*.go", "go.mod", "go.sum"]
+  }
+
+  // Output
+  client: filesystem: "./output": write: {
+    contents: actions.build.output
+  }
+
+  actions: {
+    // Alias on source
+    _source: client.filesystem.".".read.contents
+
+    // Build go binary
+    build: go.#Build & {
+      source: _source
+    }
+  }
+}
+```
+
+To build a Go binary, just run `dagger do build`. It should then take some time to
+install dependencies, build the binary and output it.
+
+Here's an example of a run
+
+```shell
+[✔] actions.build.container                                               338.0s
+[✔] client.filesystem.".".read                                              0.1s
+[✔] actions.build.container.export                                          0.1s
+[✔] client.filesystem."./output".write                                      0.2s
+```
+
+Indeed, Dagger has an ephemeral cache so if you rerun it, that shouldn't take
+that long.
+
+```shell
+[✔] actions.build.container                                                 2.9s
+[✔] client.filesystem.".".read                                              0.0s
+[✔] actions.build.container.export                                          0.0s
+[✔] client.filesystem."./output".write                                      0.1s
+```
+
+But if you stop the Buildkit daemon and remove its storage volume, cache will be lost and
+all actions will be executed again.
+
+:::info
+Now we have seen how ephemeral cache works, let's continue to understand how
+store cache in your local filesystem, so you can clean your Buildkit daemon without
+losing all your CI's cache.
+:::
+
+## Persistent cache in your local filesystem
+
+To store cache in your local filesystem, you don't need much effort : just
+run `dagger do build --cache-from type=local,mode=max,dest=<output folder>`
+
+:::tip
+Using `mode=max` argument will cache **all** layers from intermediate
+steps, with is really useful in the context of Dagger where you will have
```

Comment:
```suggestion
steps, which is really useful in the context of Dagger where you will have
```

Reply:
Nice catch, fixed in https://github.com/dagger/dagger/pull/2035/commits/1651d359cfc25c6d69b505ae6103eeb90a06dc66